_po4a-tools/po4a-update-templates.sh

```diff
@@ -64,7 +64,7 @@ while IFS= read -r -d '' doc ; do
         if [ $ext == yml ] ; then
             FILE_FORMAT=yaml
             OPTION="skip_array"
-        elif [[ $ext == html || "$filename" == *'-index' ]] ; then
+        elif [[ $ext == html || "$filename" == *'-index' ]] ; then # '-index.md' has a markdown extension but is actually html and should be processed as such by po4a
             FILE_FORMAT=xml
             OPTION="ontagerror=warn"
         elif [[ $ext == md && "$filename" != *'-index' ]] ; then
```

Comment:
The duplication of `*-index` checking isn't needed.
```suggestion
        elif [[ $ext == md ]] ; then
```

Reply:
True, it's enough if the `elif` above it catches it, thanks!